src/components/avatar/avatar.tsx

```diff
@@ -91,20 +128,17 @@ export const EuiAvatar: FunctionComponent<EuiAvatarProps> = ({
     ? '#FFFFFF'
     : '#000000';
 
-  const avatarStyle = {
-    backgroundImage: imageUrl ? `url(${imageUrl})` : 'none',
-    backgroundColor: assignedColor,
-    color: textColor,
-  };
-
   return (
-    <div
-      className={classes}
-      style={avatarStyle}
-      aria-label={name}
-      title={name}
-      {...rest}>
+    <div className={classes} aria-label={name} title={name} {...rest}>
       {optionalInitial}
+      {styles}
+      <style jsx>{`
+        .euiAvatar {
```

Comment:
Just to check, I made sure overwriting worked correctly here. Anything we write in here will correctly overwrite the imported Sass selector of `.euiAvatar` from the actual `.euiAvatar` file 👍 

This selector load order is pretty weird though.

![image](https://user-images.githubusercontent.com/324519/60942025-f1ad3d00-a295-11e9-81b3-f95ced2f5201.png)